secantTests/Util/WalletStorageTests.swift

```diff
@@ -21,31 +21,25 @@ extension KeyStoringError {
     }
 }
 
-class RecoveryPhraseStorageTests: XCTestCase {
+class WalletStorageTests: XCTestCase {
     let birthday = BlockHeight(12345678)
     let seedPhrase = "one two three"
     let language = MnemonicLanguageType.english
-    var storage: RecoveryPhraseStorage?
+    var storage = WalletStorage()
 
     override func setUp() {
         super.setUp()
-        storage = RecoveryPhraseStorage()
-        deleteData(forKey: RecoveryPhraseStorage.Constants.zcashStoredWallet)
+        deleteData(forKey: WalletStorage.Constants.zcashStoredWallet)
     }
     
-    override func tearDown() {
-        super.tearDown()
-        storage = nil
-    }
-    
-    func testWalletStoredSucessfuly() throws {
+    func testWalletStoredSuccessfuly() throws {
         do {
-            try storage?.importRecoveryPhrase(bip39: seedPhrase, birthday: birthday)
-            guard let data = data(forKey: RecoveryPhraseStorage.Constants.zcashStoredWallet) else {
+            try storage.importWallet(bip39: seedPhrase, birthday: birthday)
```

Comment:
I see we've removed the interactor but we're still relying on real data here. That makes these more or less integration tests.  Do you think it makes sense to wrap the Security dependency so that we can control it and make these unit tests?

Reply:
Are you referring to mocking the underlying keychain calls?

Reply:
Then security framework has static functions we are calling here and relying on in tests.  I'm suggesting we wrap that functionality so we don't have to rely on the real data to write tests.

Reply:
agreed. 

I don't think it's too much work to mock those two calls.

Reply:
I must say I'm a bit confused... this PR is "only" naming refactor and protocol oriented code -> value oriented code change.. The previous RecoveryPhraseStorage (including tests) has been approved already... these tests are not new, not different. Can we merge this and iterate over tests as a separated ticket? 

I agree we need to test live keychain implementation because this is a bare bone of the wallet. If this component fails, nothing will work. If we want to mock keychain and fake it, I'm ok, let's do that but don't block this one now.

@pacu @adamstener

Reply:
Resolving and deferring to issue
https://github.com/zcash/secant-ios-wallet/issues/231